src/index.ts

```diff
@@ -406,7 +406,11 @@ function HalJsonVuex (store: Store<Record<string, State>>, axios: AxiosInstance,
   function setLoadPromiseOnStore (uri: string, loadStoreData: Promise<StoreData> | null = null) {
     const promise: SerializablePromise<StoreData> = loadStoreData || Promise.resolve(store.state[opts.apiName][uri])
     promise.toJSON = () => '{}' // avoid warning in Nuxt when serializing the complete Vuex store ("Cannot stringify arbitrary non-POJOs Promise")
-    store.state[opts.apiName][uri]._meta.load = promise
+
+    store.commit('setLoadPromise', {
```

Comment:
Have you read the comment on line 399? I think I had to do it thisway, otherwise I would run into problems... Change LGTM, but I'm unsure because of that comment